license.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
@battaglr From [here](https://help.github.com/articles/adding-a-license-to-a-repository/#including-an-open-source-license-in-your-repository): file name is LICENSE or LICENSE.md (with all caps). Para que cambias a `license.txt`?